### DISCUSSION TITLE
Any possibale that  i can short TOC?

### DISCUSSION THREAD

**Discussion by ning3739** [quarto-dev/quarto-cli#4978] [Q&A]
![Screenshot 2023-03-27 at 10 32 38 PM](https://user-images.githubusercontent.com/94179273/227904214-d5a4ed70-8a2a-4946-a74c-689ef9995bea.png)
Hi there, 
During the construction of my personal [website](https://stats-r.com/series/data%20visualization/), I discovered that my Table of Contents was overly lengthy and unappealing in appearance. As a result, I am wondering if there is a way to reduce the size of this section without making it too conspicuous? 
Thank you so much !!!

**Comment by mcanouil** [comment]
What do you mean by "size"? Font size, height, number of items, something else? 🤔
